src/components/shared/Entity/Error/index.tsx

```diff
@@ -15,31 +15,19 @@ interface Props {
 
 function EntityError({ logToken, error, title, draftId }: Props) {
     return (
-        <Box sx={{ width: '100%' }}>
-            <Alert
-                icon={false}
-                sx={{
-                    'width': '100%',
-                    '& .MuiAlert-message': { width: '100%' },
-                }}
-                severity="error"
-            >
-                <AlertTitle>
-                    <FormattedMessage id={title} />
-                </AlertTitle>
-                <Stack direction="column" spacing={2}>
-                    <Box
-                        sx={{
-                            overflow: 'auto',
-                        }}
-                    >
-                        <Error error={error} hideTitle={true} />
-                        {draftId ? <DraftErrors draftId={draftId} /> : null}
-                    </Box>
-                    <ErrorLogs logToken={logToken} />
-                </Stack>
-            </Alert>
-        </Box>
+        <HeaderSummary severity="error" title={title}>
```

Comment:
I think we'll eventually need a summary for warnings, info, success, etc. and not just errors. So a new component.